sonaranalyzer-dotnet/tests/SonarAnalyzer.UnitTest/Helpers/AssertIgnoreScope.cs

```diff
@@ -31,20 +31,24 @@ public sealed class AssertIgnoreScope : IDisposable
     {
         public AssertIgnoreScope()
         {
+#if NETFRAMEWORK // Not needed on .Net Core
```

Comment:
I would rephrase this to "Is a NET Framework specific API"

Not needed on .NET core is a big vague (why is it not needed?)